src/content/docs/release-notes/docs-release-notes/docs-5-17-2024.mdx

```diff
@@ -0,0 +1,70 @@
+
+### Minor changes
+
+* In our [NerdGraph workflows tutorial](/docs/apis/nerdgraph/examples/nerdgraph-api-workflows/#update-workflow), added a GraphQL example to add a workflow name and two channels. 
+* Added a note that when you're [creating a custom log parsing rule](/docs/logs/ui-data/parsing/#custom-parsing) you need to paste unformatted logs into the parsing UI. 
+* Added **View session replays** to the browser [user permissions](/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions/#browser) list.
+* Fixed incorrect [Sonarqube config](/docs/infrastructure/host-integrations/host-integrations-list/sonarqube-monitoring-integration) file example.
+* Added attributes to our data dictionary for the `LlmChatCompletionMessage`, `LlmChatCompletionSummary`, and `LlmEmbedding` event types.
+* Added session replay capitalization guidance to our [style guide](/docs/style-guide/capitalization/product-capability-feature-usage).
+* Added a stronger definition of [log parsing](/docs/logs/ui-data/parsing) to our intro paragraph.
+* Added `gov-aws-api.newrelic.com` to our list of [FedRAMP-compliant endpoints](/docs/security/security-privacy/compliance/fedramp-compliant-endpoints). 
+* Added a NerdGraph mutation example showing how to [rotate an Azure client secret token](/docs/apis/nerdgraph/examples/nerdgraph-cloud-integrations-api-tutorial/#rotate-azure-token).
+* Responding to customer feedback, updated [Use your charts](/docs/query-your-data/explore-query-data/use-charts/use-your-charts) to clarify the **Numeric**, **Date**, **Duration**, and **Time to Now** chart options in the UI.
+* The [errors inbox OpenTelemetry](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-errors-inbox-page) page in the UI has a new `http.response.status_code` group message value.
+* Our [alerts notification message templates](/docs/alerts-applied-intelligence/notifications/message-templates/#escape) now support escape characters to help our customers avoid JSON-breaking characters, among other things.
+* You may have noticed our code blocks are looking a lot better. We've updated many of our codeblocks with language-specific syntax highlighting so that they're prettier and easier to read.
+* Updated code examples in the Python APM agent [data_source_generator](/docs/apm/agents/python-agent/python-agent-api/datasourcegenerator-python-agent-api/#example) and [register_data_source](/docs/apm/agents/python-agent/python-agent-api/registerdatasource-python-agent-api/#examples) API docs.
+* Expanded our [Azure log forwarding troubleshooting](/docs/logs/forward-logs/azure-log-forwarding/#troubleshoot) help.  
+
+
+### Release notes and What's New posts
+
+* What's New post for [Tabs for query builder](/whats-new/2024/05/whats-new-05-16-query-tabs).
```

Comment:
@paperclypse Since these are not even close to complete sentences, maybe you don't need periods after each. It looks like you've been using periods in previous sets of release notes, so you should probably stick with them.